src/ol/format/GeoJSON.js

```diff
@@ -162,6 +162,8 @@ class GeoJSON extends JSONFeature {
     if (crs) {
       if (crs['type'] == 'name') {
         projection = getProjection(crs['properties']['name']);
+      } else if (crs['type'] === 'EPSG') {
```

Comment:
This syntax was removed in draft 6 of the community standard 11 years ago.  I’m curious where you are seeing it used.

Reply:
> This syntax was removed in draft 6 of the community standard 11 years ago. I’m curious where you are seeing it used.

Unfortunately, it still exists in few of Poland's governments WFS's :|